scalableminds/webknossos
# Backup old datasource-properties upon change

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- edit datasource properties via wK (i.e. using the dataset edit page)
- backup log text file should be created

### Issues:
- fixes #4530 

------
